ilastik/applets/wsdt/wsdtGui.py

```diff
@@ -124,50 +124,55 @@ def populate_channel_menu(*args):
-        self.superpixel_size_box = superpixel_size_box
-
-        preserve_pmaps_box = QCheckBox()
-        configure_update_handlers( preserve_pmaps_box.toggled, op.PreserveMembranePmaps )
-        preserve_pmaps_box.setToolTip("Preserve thin structures. Use that option when some of your foreground objects have long and thin parts")
-        drawer_layout.addLayout( control_layout( "Preserve Thin Structures", preserve_pmaps_box ) )
-        self.preserve_pmaps_box = preserve_pmaps_box
+        min_size_box = QSpinBox()
+        min_size_box.setMinimum(0)
+        min_size_box.setMaximum(1000000)
+        configure_update_handlers( min_size_box.valueChanged, op.MinSize )
+        min_size_box.setToolTip("Mininum size of the watershed segments")
+        drawer_layout.addLayout( control_layout( "Min Boundary Size", min_size_box ) )
+        self.min_size_box = min_size_box
+
+        sigma_seeds_box = QDoubleSpinBox()
+        sigma_seeds_box.setDecimals(1)
+        sigma_seeds_box.setMinimum(0.0)
+        sigma_seeds_box.setMaximum(10.0)
+        sigma_seeds_box.setSingleStep(0.1)
+        configure_update_handlers( sigma_seeds_box.valueChanged, op.SigmaSeeds )
+        sigma_seeds_box.setToolTip("Smooth the watershed seed map with this sigma")
+        drawer_layout.addLayout( control_layout( "Seeds Smooth", sigma_seeds_box ) )
+        self.sigma_seeds_box = sigma_seeds_box
+
+        alpha_box = QDoubleSpinBox()
+        alpha_box.setDecimals(1)
+        alpha_box.setMinimum(0.0)
+        alpha_box.setMaximum(10.0)
+        alpha_box.setSingleStep(0.1)
+        configure_update_handlers( alpha_box.valueChanged, op.Alpha )
+        alpha_box.setToolTip("Used to blend boundaries and the distance transform in order to obtain the watershed weight map")
+        drawer_layout.addLayout( control_layout( "Alpha", alpha_box ) )
+        self.alpha_box = alpha_box
+
+        sigma_weights_box = QDoubleSpinBox()
+        sigma_weights_box.setDecimals(1)
+        sigma_weights_box.setMinimum(0.0)
+        sigma_weights_box.setMaximum(10.0)
+        sigma_weights_box.setSingleStep(0.1)
+        sigma_weights_box.setValue(2.0) # Default
+        configure_update_handlers( sigma_weights_box.valueChanged, op.SigmaSeeds )
+        sigma_weights_box.setToolTip("Smooth the watershed weight map with this sigma")
+        drawer_layout.addLayout( control_layout( "Weight Smooth", sigma_weights_box ) )
+        self.sigma_weights_box = sigma_weights_box
+
+        apply_nonmax_suppression_box = QCheckBox()
+        configure_update_handlers( apply_nonmax_suppression_box.toggled, op.ApplyNonmaxSuppression )
+        apply_nonmax_suppression_box.setToolTip("Filter out seeds. Use that option when some of your foreground objects have long and thin parts.")
+        drawer_layout.addLayout( control_layout( "Nonmax Suppression", apply_nonmax_suppression_box ) )
+        self.apply_nonmax_suppression_box = apply_nonmax_suppression_box
```

Comment:
I don't think we need to expose all parameters to the user here.
`sigma_weights` and `sigma_seeds` can be unified to `sigma`.
And I don't think that we need to expose `apply_nonmax_suppression`, it can just be enabled by default.
The only downside to using non-max suppression for the seeds is a bit longer run-time + nifty dependency (which is not an issue here.)